### PR TITLE
Move @{} notation translation from js to R

### PR DESCRIPTION
**Pull request by hideaki**
### Description
Move @{} notation translation from js to R.

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

**Comment by hideaki** [comment]
Tested following on Collab Server too
- Slides with Athena - Refresh and Interactive session
- Dashboard with Mongo - Refresh and Interactive session

File: tests/testthat/test_system.R [hunk 1]
```diff
@@ -129,25 +129,31 @@ test_that("countycode", {
...
-  res <- glue::glue("{v*}", .transformer=bigquery_glue_transformer)
+  exploratory_env <- new.env()
+  exploratory_env$v <- c(1,2,3)
+  res <- glue_exploratory("@{ v }", .transformer=bigquery_glue_transformer)
   expect_equal(as.character(res), "1, 2, 3")
-  v <- c("a","b","c")
-  res <- glue::glue("{v*}", .transformer=bigquery_glue_transformer)
-  expect_equal(as.character(res), '"a", "b", "c"')
+  exploratory_env$v <- c("a","b","c")
+  res <- glue_exploratory("@{ `v` }", .transformer=sql_glue_transformer)
```
Comment (kei51e) [inline tests/testthat/test_system.R]:
```suggestion
   res <- glue_exploratory("@{ `v` }", .transformer=bigquery_glue_transformer)
```

I think this is testing bigquery_glue_transformer, not sql_glue_transformer.

Comment (hideaki) [reply tests/testthat/test_system.R]: Thank you for catching this. Corrected it.

File: tests/testthat/test_system.R [hunk 2]
```diff
@@ -129,25 +129,31 @@ test_that("countycode", {
...
-  res <- glue::glue("{v*}", .transformer=odbc_glue_transformer)
+test_that("sql_glue_transformer", {
+  exploratory_env <- new.env()
+  exploratory_env$v <- c(1,2,3)
+  res <- glue_exploratory("@{ v }", .transformer=sql_glue_transformer)
   expect_equal(as.character(res), "1, 2, 3")
-  v <- c("a","b","c")
-  res <- glue::glue("{v*}", .transformer=odbc_glue_transformer)
+  exploratory_env$v <- c("a","b","c")
+  res <- glue_exploratory("@{ `v` }", .transformer=sql_glue_transformer)
```
[the remark this answers is not in the document]
Comment (hideaki) [reply tests/testthat/test_system.R]: Added a case with realistic SQL.

**Pull request merged**